src/Filter/FilterInterface.php

```diff
@@ -27,144 +27,113 @@ interface FilterInterface
     /**
      * Apply the filter to the QueryBuilder instance.
      *
-     * @param string  $alias
-     * @param string  $field
      * @param mixed[] $value
      */
-    public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $value);
+    public function filter(ProxyQueryInterface $queryBuilder, string $alias, string $field, $value): void;
```

Comment:
BTW I think the parameter `alias` only applies to ORM, maybe we can remove it, but it means to modify all filters that use it in the SonataDoctrineORMBundle.

Reply:
Should we create an issue about this ?

Reply:
Yes we should. IMO all the things we discover on those PR that are not easy to change should deserve a separate issue with 4.0 milestone so other people can contribute or atleast we can track the remaining work to do

Reply:
I agree. These PR are a good way to review all our code and create some issue/todo.

We can add 4.0 milestone to at least discuss about it. Then if a issue ask to much work, it can be delay to 5.0.

Im still without computer so I let you the issue creation.

Reply:
https://github.com/sonata-project/SonataAdminBundle/issues/6269 done!